system-test/read-rows.ts

```diff
@@ -133,18 +133,6 @@ describe('Bigtable/Table', () => {
     let stub: sinon.SinonStub;
 
     beforeEach(() => {
-      clock = sinon.useFakeTimers({
-        toFake: [
-          'setTimeout',
-          'clearTimeout',
-          'setImmediate',
```

Comment:
It's possible you could just remove setImmediate and clearImmediate here (and also probably nextTick) and that would also fix it. I seem to remember having troubles with this same issue on Pub/Sub, now that you mention it, because they started hooking these by default.

Reply:
[here](https://github.com/googleapis/nodejs-pubsub/pull/2024/files#diff-2cf01d03222a34294066e6c173aea4831d26113df10f6a8969dc29a3f97cb467)